118 improve thyra its plugin documentation node manager generalities (#126)
massalabs/massa-docs: general-doc/decentralized-web/thyra.rst

.. _general-thyra:

Thyra
=====

The project
-----------

Thyra is a secured gate to Massa blockchain and pure decentralization.
Developped by Massa Labs, this application provides a user-friendly way of accessing,
using and building on the blockchain while preventing users from possible attack or hack.
Indeed, because Thyra is installed on your own device there is absolutely nothing
and no one between you and the blockchain. Security and decentralization are the app's pillars.

Concretely, Thyra enables every user to have a custom experience with the blockchain.
Once the app installed, users will be able to activate specific functionalities in a plug-and-play mode
that will provide them what they need. For instance, you will be able to run a distant node in a couple of clicks

> or if you prefer you will be able to create and publish a website on the blockchain.

Reviewer: ```suggestion
Thyra can also be used to create and publish websites on the blockchain.
```

> These functionalities are, technically speaking, plugins that can be developed by all developers that wish to.

Reviewer: ```suggestion
These functionalities are, technically speaking, plugins that can be developed by anyone.
```

> All details about how you can contribute will be added very soon.

Reviewer: ```suggestion
Technical details on how to develop plugins will be added very soon.
```
